### PR TITLE
Docs: Improving the install with `keadm` documentation

### PR DESCRIPTION
**Pull request by 1Shubham7**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs update

this is one among series of PRs planned to improve and rewrite some parts of documentation.

Works towards solving #533

### CODE REVIEW

**Comment by 1Shubham7** [comment]
@fisherxu @wbc6080, I need to know whether we should be using "Keadm" or "keadm" consistently in the docs.

File: docs/setup/install-with-keadm.md
Comment (fujitatomoya) [inline docs/setup/install-with-keadm.md:10]: i think this is not limitation but more like condition. how about this?

```suggestion
## Prerequisite
```

File: docs/setup/install-with-keadm.md
Comment (fujitatomoya) [inline docs/setup/install-with-keadm.md:25]: we are not tracking this example version, but v1.12.1 seems pretty old, can we instead refer to v1.16.2 at this time?

```suggestion
    wget https://github.com/kubeedge/kubeedge/releases/download/v1.16.2/keadm-v1.16.2-linux-amd64.tar.gz
    tar -zxvf keadm-v1.16.2-linux-amd64.tar.gz
    cp keadm-v1.16.2-linux-amd64/keadm/keadm /usr/local/bin/keadm
```

File: docs/setup/install-with-keadm.md
Comment (fujitatomoya) [inline docs/setup/install-with-keadm.md:40]: so the `CloudCore` is the correct spelling? i was thinking those are `cloudcore` and `edgecore`.

File: docs/setup/install-with-keadm.md
Comment (fujitatomoya) [inline docs/setup/install-with-keadm.md:167]: same here, i thought that is `edgecore`, that is the question here.

**Review by fujitatomoya — COMMENTED** [review]
overall, ltgm. maybe we can ask for maintainers about names of terminologies.

File: docs/setup/install-with-keadm.md
Comment (1Shubham7) [reply docs/setup/install-with-keadm.md:40]: at some places it was cloudcore and at some it was Cloudcore. should it be "cloudcore"?

File: docs/setup/install-with-keadm.md
Comment (1Shubham7) [reply docs/setup/install-with-keadm.md:25]: I agree!

File: docs/setup/install-with-keadm.md
Comment (wbc6080) [inline docs/setup/install-with-keadm.md:16]: The component name here is `Keadm`, but `keadm` is used when executing such as `keadm join` on the command line and installing packages

File: docs/setup/install-with-keadm.md
Comment (fujitatomoya) [reply docs/setup/install-with-keadm.md:40]: oh that was me misunderstanding, thanks for clarification!

**Review by fujitatomoya — APPROVED** [review]
lgtm

**Review by wbc6080 — COMMENTED** [review]
/lgtm
Thanks for your contribution
cc @Shelley-BaoYue @fisherxu

File: docs/setup/install-with-keadm.md
Comment (Shelley-BaoYue) [inline docs/setup/install-with-keadm.md:31]:
```suggestion
  docker run --rm kubeedge/installation-package:v1.17.0 cat /usr/local/bin/keadm > /usr/local/bin/keadm && chmod +x /usr/local/bin/keadm
```

File: docs/setup/install-with-keadm.md
Comment (Shelley-BaoYue) [inline docs/setup/install-with-keadm.md:92]:
```suggestion
1. Set flags `--set key=value` for CloudCore helm chart could refer to [KubeEdge CloudCore Helm Charts README.md](https://github.com/kubeedge/kubeedge/blob/master/manifests/charts/cloudcore/README.md).
```

File: docs/setup/install-with-keadm.md
Comment (Shelley-BaoYue) [inline docs/setup/install-with-keadm.md:292]:
```suggestion
    Then set all the iptables for multi CloudCore instances to every node that apiserver runs. The cloudcore ips and tunnel ports should be obtained from the configmap above.
```

File: docs/setup/install-with-keadm.md
Comment (Shelley-BaoYue) [inline docs/setup/install-with-keadm.md:6]:
```suggestion
Keadm is used to install the cloud and edge components of KubeEdge. It does not handle the installation of Kubernetes and its [runtime environment](https://kubeedge.io/docs/setup/prerequisites/runtime).
```

How about add a link with runtime? or we can put it at ##prerequisite?

Comment (1Shubham7) [reply docs/setup/install-with-keadm.md]: I have added the link with runtime. I don't think there's any point putting it at prerequisite.

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

**Comment by 1Shubham7** [comment]
@wbc6080 @fujitatomoya there have been some changes after you approved. Please approve this again.

**Review by fujitatomoya — APPROVED** [review]
thanks for iterating and being responsive. looks good to me.

**Comment by fujitatomoya** [comment]
i will leave this to @Shelley-BaoYue @wbc6080

**Review by wbc6080 — COMMENTED** [review]
/lgtm

**Comment by 1Shubham7** [comment]
@wbc6080 , @Shelley-BaoYue please approve this so that the PR gets merged. Also checkout the other improvement PRs that I have made :)

**Review by Shelley-BaoYue — COMMENTED** [review]
/approve

**Pull request merged**
